packages/competence/README.md

```
@@ -351,18 +358,21 @@ raw_team[category]    = Σ ( grade_weight(cumulative_grade[c])  × relevancy(c,
 raw_manager[category] = Σ ( grade_weight(manager_grade[c])     × relevancy(c, stageLevel) )
 ```
 
-**2. Category score** (integer, typically 0–130):
+**2. Category score** (integer, typically 0–130), **renormalized to the evaluator types that actually participated.** A source participates iff its round completed (`selfEvaluationCompleted` / `teamEvaluationCompleted` / `managerEvaluationCompleted`) — so a round that was never requested (no team assigned) or was waived by a Supervisor (a stalled self round; see [Evaluation Status Lifecycle](#evaluation-status-lifecycle)) is *excluded* from the calculation rather than counted as a zero, and no longer depresses the score:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)